### ISSUE TITLE
Configure Name dialog should indicate CBOR validity

### ISSUE THREAD

**Issue by JeremyRand** [namecoin/electrum-nmc#314]
It would be nice if the Configure Name dialog showed whether the value being entered is valid CBOR, so that the user doesn't accidentally submit a broken update.  Since CBOR is not part of the Python standard library, we should use Qt's CBOR parsing functionality.

Blocked by https://github.com/namecoin/electrum-nmc/issues/54
